createx/src/components/index.js

```diff
@@ -5,4 +5,5 @@ export { default as ProblemStatements } from './ProblemStatements/index.jsx';
 export { default as Home } from './Home/index.jsx';
 export { default as Contact } from './Contact/index.jsx';
 export { default as SubmissionPortal } from './SubmissionPortal/index.jsx';
-export { default as Sponsors } from './Sponsors/index.jsx';
\ No newline at end of file
+export { default as Sponsors } from './Sponsors/index.jsx';
+export { default as Navbar } from './Navbar/index.jsx';
```

Comment:
Now that we have design system components so move all pages components to a `pages` direcotry and keep Button and Navbar and other design system components here.